app/assets/stylesheets/avalon.css.scss

```diff
@@ -39,12 +39,7 @@
   }
 }
 
-a:not(.btn) {
-  color: $linkColor;
-  &:hover { color: $linkColor; }
-  &:active { color: $linkColor; }
-  &:visited { color: $linkColor; }
-}
+
```

Comment:
Actually looking closer at it I don't want to fix all the spacing issue in this one without talking to Chris first.